.github/workflows/CI.yaml

```diff
@@ -299,6 +299,8 @@ jobs:
           MAILJET_API_SECRET: ${{ secrets.MAILJET_API_SECRET }}
           NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST: "dummy.mail@dataland.com"
           NOTIFICATION_RECEIVERS_CC_BULK_DATA_REQUEST: "dummy.mail2@dataland.com"
+          NOTIFICATION_RECEIVERS_DATA_OWNERSHIP_REQUEST: "dummy.mail@dataland.com"
```

Comment:
Do we need this, because if we don't set it our system crashes?

Reply:
The code did not look like that it would crash. But I remember we once had this issue. I can test it if you want. But this would be working anyways.

Reply:
ok, i also remembered something in this direction. then i will close this here